qiskit/ignis/verification/accreditation/circuits.py

```diff
@@ -14,410 +14,131 @@
-                    if len(cz_gate_aux) != 1:
-                        while cz_gate_aux[0][0] == band:
-                            circ.cz(q_reg[int(cz_gate_aux[0][1])],
-                                    q_reg[int(cz_gate_aux[0][2])])
-                            cz_gate_aux = np.delete(cz_gate_aux, (0), axis=0)
-                    circ.barrier()
-                    band = band+1
-
-            # Map the quantum measurement to the classical bits
-            circ.measure(q_reg, c_reg)
-
-            circuit_list[k].append(circ)
-            postp_list[k].append(postp)
-
-        else:  # Generating a trap circuit
-
-            # Create a Quantum Register with n_qb qubits.
-            q_reg = QuantumRegister(n_qb, 'q')
-            # Create a Classical Register with n_qb bits.
-            c_reg = ClassicalRegister(n_qb, 's')
-            # Create a Quantum Circuit acting on the q register
-            circ = QuantumCircuit(q_reg, c_reg)
-
-            # Generate 1-qubit gates for trap
-            gate_trap = routine_two(n_qb, m_bands, cz_gate)
-            # Append one-time-pad to gates in trap circuit
-            gate_trap_qotp, postp = routine_one(gate_trap, cz_gate, 1)
-
-            # Increase the size of conn for further use in while loop
-            cz_gate_aux = np.vstack([cz_gate, [0, 0, 0]])
-
-            band = 0
-
-            for j in range(len(gate_trap_qotp[0])):
-                for i in range(n_qb):
-                    circ.u3(gate_trap_qotp[i][j][0],
-                            gate_trap_qotp[i][j][1],
-                            gate_trap_qotp[i][j][2],
-                            i)
-                # This places the cZ gates
-                if j % 4 == 3 and j != len(gate_trap_qotp[0])-1:
-                    circ.barrier()
-                    if len(cz_gate_aux) != 1:
-                        while cz_gate_aux[0][0] == band:
-                            circ.cz(q_reg[int(cz_gate_aux[0][1])],
-                                    q_reg[int(cz_gate_aux[0][2])])
-                            cz_gate_aux = np.delete(cz_gate_aux, (0), axis=0)
-                    circ.barrier()
-                    band = band+1
-
-            # Map the quantum measurement to the classical bits
-            circ.measure(q_reg, c_reg)
-
-            circuit_list[k].append(circ)
-            postp_list[k].append(postp)
-
-    return circuit_list, postp_list, v_zero
-
-
-def accreditation_parser(target_circuit):
-    """Converts an input quantum circuit to lists representing the input
-
-    Args:
-        target_circuit (QuantumCircuit): Quantum circuit consisting of
-        cZ gates and single qubit gates, followed by Pauli-Z measure-
-        ments on all qubits
-
-    Returns:
-        gates_target (list): A 2D list of all 1-qubit gates in the
-            target circuit
-        cz_gate (list): list of all cz gate in target circuit
+    def targetCircuit(self, targetcirc, twoqubitgate='cz', coupling_map=None):
```

Comment:
While we use camel case for classes, for functions/methods in rest of qiskit we use snake case
```suggestion
    def target_circuit(self, target_circ, two_qubit_gate='cz', coupling_map=None):
```